### PR TITLE
fix: remove orgs from token permission list

### PR DESCRIPTION
**Pull request by glinton**
Closes #influxdata/idpe#5940

Error message received when creating all access token:
```json
{
  "code": "forbidden",
  "message": "permission read:orgs/03cf84c67a620000/orgs is not allowed: read:orgs/03cf84c67a620000/orgs is unauthorized"
}
```

Resource `/orgs/{orgID}/orgs` isn't defined, removed the `orgs` entry from the authorizations portion.

Probably works.

### CODE REVIEW

**Comment by desa** [comment]
I think we do want this permission
```
  {	
    action: 'read',
    resource: {type: 'orgs', id: orgID},
  }
```

**Comment by glinton** [comment]
then something deeper doesn't work, it seems (i assume) that it requests `/orgs/{org_id}/{resource_type}` and `/orgs/id/orgs` doesn't exist. i'll try running/reproducing this locally to verify what gets requested.

**Comment by desa** [comment]
I think that
```
  {	
    action: 'read',
    resource: {type: 'orgs', id: orgID},
  }
```
should create the permission `/orgs/{orgid}`

File: ui/src/authorizations/utils/permissions.ts [hunk 1]
```diff
@@ -59,6 +58,13 @@ const ensureT = (orgID: string) => (t: PermissionTypes) => {
           resource: {type: t, orgID},
         },
       ]
+    case 'orgs':
```
Comment (desa) [inline ui/src/authorizations/utils/permissions.ts]: Can you add a  comment here noting specifically that we expect the org permission to be at `id` and not to have `orgID`?

**Pull request merged**